toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -1613,7 +1614,25 @@ func ProvisionUserSSHCerts(installChroot safechroot.ChrootInterface, username st
 	}
 	defer os.Remove(authorizedKeysTempFile)
 
-	allSSHKeys := make([]string, 0, len(sshPubKeyPaths)+len(sshPubKeys))
+	allSSHKeys := []string(nil)
+
+	if includeExistingKeys {
+		authorizedKeysFileFullPath := filepath.Join(installChroot.RootDir(), authorizedKeysFile)
+
+		exists, err := file.PathExists(authorizedKeysFileFullPath)
```

Comment:
nit: maybe rename variable to fileExists

Reply:
Fixed.